src/style-property-map.js

```diff
@@ -22,7 +22,7 @@
   StylePropertyMap.prototype.set = function(property, value) {
     var cssPropertyDictionary = propertyDictionary();
     if (!cssPropertyDictionary.isSupportedProperty(property)) {
-      throw new TypeError(property + 'is not a supported CSS property');
+      throw new TypeError(property + ' is not a supported CSS property');
```

Comment:
It's generally good practice to have different error messages if they appear more than once - that way you know exactly where to look if you see that message. So in this case you can say

"Cannot set " + property + " because it is not a supported CSS property"
"Cannot delete " + property + ... etc